### PR TITLE
[GEN-497] New Record Tab

### PR DESCRIPTION
**Pull request by VGmt**
- Added a new `Record` tab in Navigation
- Created 2 files, front and backend in `Record` folder
- moved relevant functions from the previous Backend Logic implementation in `HomeScreenLogic.js`

269B804B-B3AE-4A98-BFD2-1883938640FB_1_102_o

### CODE REVIEW

File: trk/src/Navigation/AppNavigator.js [hunk 1]
```diff
@@ -98,6 +99,38 @@ function HomeStack() {
...
         options={{ title: 'Developer Feedback', headerTitleAlign: 'center' }}
       />
+      <Stack.Screen
+        name="Feedback"
+        component={FeedbackForm}
+        options={{ title: 'Feedback Form', headerBackTitle: 'Climb Detail', headerTitleAlign: 'center' }}
+      />
+    </Stack.Navigator>
+  );
+}
```
Comment (VGmt) [inline trk/src/Navigation/AppNavigator.js:109]: Github is behaving strangely. I only **removed** `Stack.Screen`s from HomeStack(), didn't add any.

Anyway, removed the navigations to `ClimbDetail` and `GlossaryDefinitions` as they are no longer accessible from `Home`

File: trk/src/Navigation/AppNavigator.js [hunk 2]
```diff
@@ -98,6 +99,38 @@ function HomeStack() {
...
+            />
+          ),
+        })}
+      />
+      <Stack.Screen
+        name="Developer_Feedback"
+        component={DeveloperFeedbackForm}
+        options={{ title: 'Developer Feedback', headerTitleAlign: 'center' }}
+      />
       <Stack.Screen
```
Comment (VGmt) [inline trk/src/Navigation/AppNavigator.js:134]: And added them in the `RecordStack`

File: trk/src/Navigation/AppNavigator.js [hunk 3]
```diff
@@ -270,6 +303,24 @@ function AppTabs() {
...
+
+          // tabBarIcon: ({ size, focused, color }) => {
+          //   return (
+          //     <Image
+          //       style={{ width: size, height: size }}
+          //       source={require('../../assets/record.png')}
+          //     />
+          //   );
+        }}
+      />
```
Comment (VGmt) [inline trk/src/Navigation/AppNavigator.js:323]: Didn't want to add a logo that wasn't agreed upon.

**Review by redpepper-nag — APPROVED** [review]
Screenshot 2024-01-05 at 2 11 31 PM
How about this for the icon (it's the one from our figma)

**Comment by VGmt** [comment]
Sounds good. Can implement in another PR

**Review by abhipi — APPROVED** [review]

**Pull request merged**